pkg/controller/mizar/mizar-arktos-network-controller.go

```diff
@@ -41,15 +46,31 @@ import (
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog"
 	"k8s.io/kubernetes/pkg/controller"
+	"k8s.io/kubernetes/pkg/features"
 )
 
 const (
 	mizarNetworkType = "mizar"
+	mizarInternalIPStart = 20
+
+	resource_group ="mizar.com"
+	resource_version = "v1"
+	resource_vpc = "vpcs"
+	resource_subnet = "subnets"
 )
 
 var seed = rand.NewSource(time.Now().UnixNano())
 var ran = rand.New(seed)
 
+// Temporary solution for mizar VPC range cannot overlapping issue
```

Comment:
since tenant deletion is not supported yet, we may not need map based vpc range management for now. Just a suggestion for a simpler mechanism.

Reply:
> since tenant deletion is not supported yet, we may not need map based vpc range management for now. Just a suggestion for a simpler mechanism.

This is mainly used for VPC that were created outside of automated process. Also works for KCM recovery scenario.